docs/snippets/ov_extensions.cpp

```diff
@@ -55,6 +56,7 @@ class CustomOp : public ov::op::Op {
     OPENVINO_OP("CustomOp");
     OPENVINO_FRAMEWORK_MAP(onnx, "CustomOp", { {"m_mode", "mode"} }, { {"m_axis", -1} });
     OPENVINO_FRAMEWORK_MAP(tensorflow, "CustomOpV3", { {"m_axis", "axis"} }, { {"m_mode", "linear"} });
+    OPENVINO_FRAMEWORK_MAP(paddle, {"X"}, {"Out"}, "CustomOp", { {"m_mode", "mode"} }, { {"m_axis", -1} });
```

Comment:
Adding input and output names makes this API confusing. Initially, the macro was aligned with C++ API. Is this still true? Cannot find confirmation in this document. Could you give me a link?

Any other variants to make it more fluent for Paddle?

Now we don't have time to fix it in this release, but it should be recorded as an improvement possibility for the next one.

Reply:
> Adding input and output names makes this API confusing. Initially, the macro was aligned with C++ API. Is this still true? 

Yes, that's still true, for example https://github.com/openvinotoolkit/openvino/pull/16752/files/eb29a4e1642f199bc982d002f034ac580f0a822c#diff-66323682770990fd131c3d9a8016bb65100dcac4aaf23740bffb5279ffe9fb7cR157.

> Now we don't have time to fix it in this release, but it should be recorded as an improvement possibility for the next one.

CVS-110254